ctapipe/coordinates/tests/test_coordinates.py

```diff
@@ -268,3 +268,15 @@ def test_ground_frame_roundtrip():
     assert u.isclose(coord.x, back.x)
     assert u.isclose(coord.y, back.y)
     assert u.isclose(coord.z, back.z)
+
+
+def test_ground_to_eastnorth_roundtrip():
```

Comment:
Not only testing the round trip but also actual values would be good.

Reply:
done